templates/init/functions/typescript/_eslintrc

```diff
@@ -10,6 +10,8 @@ module.exports = {
     "plugin:import/warnings",
     "plugin:import/typescript",
     "google",
+    "plugin:@typescript-eslint/eslint-recommended",
```

Comment:
What is `@typescript-eslint/eslint-recommended`?  I'm having a hard time finding documentation for it.

Reply:
It's a bit buried in the documentation but you can see it [here](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md)